hack/lib.sh

```diff
@@ -33,10 +33,12 @@ HELM_VERSION=2.9.1
 KIND_VERSION=0.7.0
 KIND_BIN=$OUTPUT_BIN/kind
 KUBEBUILDER_PATH=$OUTPUT_BIN/kubebuilder
-KUBEBUILDER_BIN=$KUBEBUILDER_PATH/bin/kubebuilder
+KUBEBUILDER_BIN=$OUTPUT_BIN/kubebuilder
```

Comment:
We are having `KUBEBUILDER_PATH` and `KUBEBUILDER_BIN` here, are they both required?

Reply:
no, I will finally drop `KUBEBUILDER_PATH`